domain/src/main/assets/sample_questions.json

```diff
@@ -0,0 +1,263 @@
+{ "questions":
+[{
+  "name":"question",
+  "classifier_model_id":null,
+  "content":{
+    "html":"<p>What fraction does 'quarter' represent?&nbsp;</p>",
+    "content_id":"content"
+  },
+  "interaction":{
+    "confirmed_unclassified_answers": [],
+    "answer_groups":[
+      {
+        "rule_specs":[
+          {
+            "rule_type":"Contains",
+            "inputs":{
+              "x":"1/4"
+            }
+          }
+        ],
+        "outcome":{
+          "dest":null,
+          "feedback":{
+            "html":"<p>That's correct!</p>",
+            "content_id":"feedback_1"
+          },
+          "labelled_as_correct":true,
+          "param_changes":[
+
+          ],
+          "refresher_exploration_id":null,
+          "missing_prerequisite_skill_id":null
+        },
+        "trainingData":[
+
+        ],
+        "taggedSkillMisconceptionId":null
+      }
+    ],
+    "confirmedUnclassifiedAnswers":[
+
+    ],
+    "customizationArgs":{
+      "placeholder":{
+        "value":"Write your fraction as x/y"
+      },
+      "rows":{
+        "value":1
+      }
+    },
+    "defaultOutcome":{
+      "dest":null,
+      "feedback":{
+        "html":"",
+        "content_id":"default_outcome"
+      },
+      "labelled_as_correct":false,
+      "param_changes":[
+
+      ],
+      "refresher_exploration_id":null,
+      "missing_prerequisite_skill_id":null
+    },
+    "hints":[
+      {
+        "hintContent":{
+          "html":"<p>Hint text will appear here</p>",
+          "content_id":"hint_1"
+        }
+      }
+    ],
+    "id":"TextInput",
+    "solution":{
+      "answer_is_exclusive":false,
```

Comment:
Nit here & elsewhere: please format the JSON using a JSON formatter & update.

Reply:
Done